src/main/webapp/app/exercises/programming/shared/code-editor/layout/code-editor-grid.scss

```diff
@@ -162,17 +162,29 @@ Collapsable styling
         .editable-instruction-container__editor {
             display: none;
         }
-        .instructions__content {
-            display: none;
-        }
     }
 
     .card-second-header {
         display: none !important;
     }
 
-    .card-title span {
-        display: none !important;
+    .collapsed-instructions-card {
+        width: 50px;
+        justify-content: space-between;
+        flex-flow: column nowrap;
+        cursor: pointer;
+
+        .header {
+            writing-mode: vertical-rl;
+            margin-left: -3px;
+        }
+
+        fa-icon {
```

Comment:
I am not really sure what you can do to resolve the Codacy issue. Maybe you can extract the fa-icon outside the .collapsed-instructions-card or add an id or a css class to the fa-icon in the HTML part.